### ISSUE TITLE
Mono.Cecil Exception in classes that have a property of struct implementing IEnumerable

### ISSUE THREAD

**Issue by mikhailshilkov** [Fody/Equals#26]
```
1>MSBUILD : error : Fody: An unhandled exception occurred:
1>MSBUILD : error : Exception:
1>MSBUILD : error : Member 'Bar`1' is declared in another module and needs to be imported
1>MSBUILD : error : StackTrace:
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.LookupToken(IMetadataTokenProvider provider)
1>MSBUILD : error :    at Mono.Cecil.Cil.CodeWriter.WriteOperand(Instruction instruction)
1>MSBUILD : error :    at Mono.Cecil.Cil.CodeWriter.WriteInstructions()
1>MSBUILD : error :    at Mono.Cecil.Cil.CodeWriter.WriteResolvedMethodBody(MethodDefinition method)
1>MSBUILD : error :    at Mono.Cecil.Cil.CodeWriter.WriteMethodBody(MethodDefinition method)
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.AddMethod(MethodDefinition method)
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.AddMethods(TypeDefinition type)
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.AddType(TypeDefinition type)
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.AddTypeDefs()
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.BuildTypes()
1>MSBUILD : error :    at Mono.Cecil.MetadataBuilder.BuildModule()
1>MSBUILD : error :    at Mono.Cecil.ModuleWriter.<BuildMetadata>b__0(MetadataBuilder builder, MetadataReader _)
1>MSBUILD : error :    at Mono.Cecil.ModuleDefinition.Read[TItem,TRet](TItem item, Func`3 read)
1>MSBUILD : error :    at Mono.Cecil.ModuleWriter.BuildMetadata(ModuleDefinition module, MetadataBuilder metadata)
1>MSBUILD : error :    at Mono.Cecil.ModuleWriter.WriteModuleTo(ModuleDefinition module, Stream stream, WriterParameters parameters)
1>MSBUILD : error :    at Mono.Cecil.ModuleDefinition.Write(Stream stream, WriterParameters parameters)
1>MSBUILD : error :    at Mono.Cecil.ModuleDefinition.Write(String fileName, WriterParameters parameters)
1>MSBUILD : error :    at InnerWeaver.WriteModule() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\ModuleWriter.cs:line 18
1>MSBUILD : error :    at InnerWeaver.Execute() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:line 85
1>MSBUILD : error : Source:
1>MSBUILD : error : Mono.Cecil
1>MSBUILD : error : TargetSite:
1>MSBUILD : error : Mono.Cecil.MetadataToken LookupToken(Mono.Cecil.IMetadataTokenProvider)
```

Here is some code to reproduce:
```
namespace Assembly
{
    [Equals]
    public class Foo
    {
        public Bar<int> Prop { get; set; }
    }
}

namespace AssemblyReferenced
{
    public struct Bar<T> : IEnumerable<T>
    {
        public IEnumerator<T> GetEnumerator()
        {
            throw new NotImplementedException();
        }

        IEnumerator IEnumerable.GetEnumerator()
        {
            return this.GetEnumerator();
        }
    }
}
```

**Comment by rjasica** [comment]
Thanks for reporting this bug. I have released new nuget packaged(1.4.9). Please let me know if you find other poblems witht FodyEquals

**Comment by mikhailshilkov** [comment]
Works like charm now, thanks a lot for swift fix!
